src/app/public/modules/action-button/action-button-permalink.ts

```diff
@@ -2,10 +2,28 @@ import {
   NavigationExtras
 } from '@angular/router';
 
+/**
+ * Specifies an Angular router link with the `route` property or a direct
+ * link with the `url` property. If it provides both, the action button uses
+ * the `route` property.
+ */
 export interface SkyActionButtonPermalink {
+/**
+ * - `permalink.route?.extras?.fragment` to
+ * [`fragment`](https://angular.io/api/router/NavigationExtras#fragment)
+ * - `permalink.route?.extras?.queryParams` to
+ * [`queryParams`](https://angular.io/api/router/NavigationExtras#queryParams)
+ * - `permalink.route?.extras?.queryParamsHandling` to
+ * [`queryParamsHandling`](https://angular.io/api/router/NavigationExtras#queryParamsHandling)
+ * - `permalink.route?.commands` to
+ * [`routerLink`](https://angular.io/api/router/RouterLink)
+ */
   route?: {
```

Comment:
This is a little different structure from what we have on the current docs site. Do you think it helps to include the intro?
```
Specifies an Angular router link for the action button. This property accepts a custom object with a commands property to specify router commands and an extras property to specify Angular NavigationExtras. This property passes the following properties: 
```
...and also contain a link to the Angular site.

Reply:
✔️